internal/samsahai/rpc.go

```diff
@@ -876,3 +899,30 @@ func (c *controller) sendPullRequestTriggerReport(prTrigger *s2hv1.PullRequestTr
 		}
 	}
 }
+
+func (c *controller) sendPullRequestTestRunnerPendingReport(prQueue *s2hv1.PullRequestQueue, teamWithPR *rpc.TeamWithPullRequest) {
+	configCtrl := c.GetConfigController()
+
+	teamComp := &s2hv1.Team{}
+	err := c.getTeam(teamWithPR.TeamName, teamComp)
+	if err != nil {
+		logger.Error(err, "cannot get team", "team", teamWithPR.TeamName)
+	}
+
+	if err := c.LoadTeamSecret(teamComp); err != nil {
+		logger.Error(err, "cannot load team secret", "team", teamComp.Name)
```

Comment:
curious that if we couldn't get team here, should we return the error? or we can continue?
won't it break the code after this e.g., `teamComp.Status.Used.Credential` ?